### PR TITLE
Remove pointless extension headers

### PR DESCRIPTION
**Pull request by JoshDreamland**
So, I must have been asleep, because I _just_ noticed you guys, following polygone's example, created `implement` headers for all the extensions. Those are totally unnecessary, and in fact, some of them (Robert's) had code in them that built but was never run.

I noticed this when I went to fix the problem with paths. Apparently, polygone commented them out sometime around 14BC because the extension system is broken. I created this pull request while fixing the extension system so that you all (A) have a chance to make sure this works before it goes upstream and (B) realize that if something in the extensions framework seems useless, it probably isn't needed.

The only thing this pull request does is delete those headers and the referring lines from the About.ey files. Before I realized Robert's code in two of his `implement.h` files was totally unused, I migrated them to appropriate source files, and so I left them in the commit.

I have tested this request with the default extensions for the ability to compile. Since no code changed, I can't imagine a change in behavior, but feel free to test to your hearts' respective contents.

### CODE REVIEW

**Comment by JoshDreamland** [comment]
I apparently closed this PR forever ago, presumably by replying using tab + enter. This change is still useful.

**Comment by RobertBColton** [comment]
@JoshDreamland some additional documentation on this could be useful. I think I still don't really understand the importance of implements. I believe it is necessary to add build-in locals? So for example, if you wanted to make an official 3D extension, you would use it to add a z local? I believe this is required then to add the locals for the GMS Physics Compatibility extension.

I was also under the impression that implements is required to get a sort of static initialization method for an extension, say to set up handles to an API like DirectShow or the Media Control Interface (used for CD functions in my one extension). This is why I was also under the impression it was how to go about getting a system update method for an extension that is called automatically, i.e. to update particle and run a physics timestep.

It would help if you could clarify this. We could use better guides on how to build extensions honestly.

File: ENIGMAsystem/SHELL/Universal_System/Extensions/Paths/path_functions.cpp [hunk 1]
```diff
@@ -60,6 +60,12 @@ namespace enigma_user
 
 void path_start(unsigned pathid, cs_scalar speed, unsigned endaction, bool absolute)
 {
+    #ifndef PATH_EXT_SET
+        return;
+    #endif
+
+    return;  //function can cause crashes atm, until extension variables fixed
```
Comment (fundies) [inline ENIGMAsystem/SHELL/Universal_System/Extensions/Paths/path_functions.cpp]: Are these actually broken still? There is no early return in our current version and I haven seen a report of crashing.

**Comment by JoshDreamland** [comment]
Looks as though the deletions have already happened in another commit. Can someone confirm that and close this?

**Comment by JoshDreamland** [comment]
Thanks for checking that. I was pretty sure this was good.

**Pull request closed without merging**